packages/consumption/test/modules/requests/itemProcessors/readAttribute/ReadAttributeRequestItemProcessor.test.ts

```diff
@@ -353,16 +353,16 @@ describe("ReadAttributeRequestItemProcessor", function () {
         });
 
         test("returns an error when the given Attribute id belongs to a peer Attribute", async function () {
-            const peer = CoreAddress.from("id1");
+            const sender = CoreAddress.from("Sender");
 
             const peerAttributeId = await ConsumptionIds.attribute.generate();
```

Comment:
While you're at it, declare the variable "peerSharedAttributeId" here instead of "peerAttributeId" please.